pkgs/applications/plasma-mobile/tokodon.nix

```diff
@@ -34,14 +34,14 @@
 libsForQt5.mkDerivation rec {
   pname = "tokodon";
 
-  version = "22.11.2";
+  version = "23.01.0";
```

Comment:
Woops, it seems in the end *maybe* there is something to do. I hadn't noticed this change when writing that earlier frustration.

The releases are now (and were since 22.11.2 it seems) handled outside of gear packages

 - https://download.kde.org/stable/tokodon/

The general policy is to prefer the source tarball releases from upstreams.

Reply:
> The general policy is to prefer the source tarball releases from upstreams.

Whose policy?

> Maybe we ignore the problem for now and wait for KDE Gear 23.04, since **anyway** the whole future of the package set needs to be reconsidered?

I doubt it would be added to KDE Gear if it wasn't even part of Plasma Mobile Gear. But I definitely wouldn't move the entire expression yet.

Reply:
> Whose policy?

Nixpkgs, NixOS.

> > Maybe we ignore the problem for now and wait for KDE Gear 23.04, since **anyway** the whole future of the package set needs to be reconsidered?
> 
> I doubt it would be added to KDE Gear if it wasn't even part of Plasma Mobile Gear. 

[It was part of Plasma Mobile Gear 22.11](https://download.kde.org/stable/plasma-mobile/22.11/), and it started being included in 21.06. What was missing was the `.2` upgrade, which fixed major issues.

Though really what I meant wasn't that 23.04 would change the status of Tokodon being part of a Gear package, but that we can delay deciding what happens with Tokodon to when Plasma Mobile Gear is torn down.

Reply:
> we can delay deciding what happens with Tokodon to when Plasma Mobile Gear is torn down

Makes sense to me.